core/model/ft.go

```diff
@@ -0,0 +1,30 @@
+package model
+
+type CreateFTReq struct {
+	DID        string  `json:"did"`
+	FTName     string  `json:"ftname"`
+	FTCount    int     `json:"ftcount"`
+	TokenCount float64 `json:"tokencount"`
+}
+
+type TransferFTReq struct {
+	Receiver   string `json:"receiver"`
+	Sender     string `json:"sender"`
+	FTName     string `json:"FTName"`
```

Comment:
It is better to follow the json naming convention as ft_name and ft_count

Reply:
changed to `ft_name and ft_count`.